nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.pipewire.wireplumber;
+in
+{
+  meta.maintainers = [ maintainers.k900 ];
+
+  options = {
+    services.pipewire.wireplumber = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "A modular session / policy manager for PipeWire";
+      };
+
+      package = mkOption {
```

Comment:
Yeah, you can pass e.g. `overlays = [ (final: prev: { wireplumber = prev.wireplumber.overrideAttrs (attrs: rec { version = "…"; src = "…"; }); }) ]` to nixpkgs (or use `nixpkgs.overlays` option if you do not override the `nixpkgs.pkgs` option) and it will replace wireplumber in `pkgs` passed to the modules.

Reply:
Not really, but pipewire-media-session has it, so I thought it should be here for consistency as well. Also, wireplumber is (sort of) a library, so it's possible other things will start depending on it at some point.

Reply:
Is overlays > `package` options in modules a documented policy nowadays? If so, it makes sense to not have this option here at all and mark it deprecated in the pipewire module as well.

Reply:
It always felt to me that `package` options are cargo cult. They should only be necessary when overriding the package globally would cause rebuilds of the system (e.g. in case of systemd).